infrastructure/ArmTemplates/storage-deploy.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storageAccount_name": {
+            "defaultValue": "diagsourcewatcher",
+            "type": "string"
+        },
+        "location": {
+            "type": "string"
```

Comment:
Create default with `resourceGroup().location`. This will default to location of the resource group. https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#resourcegroup